gnat2goto/driver/tree_walk.adb

```diff
@@ -3712,6 +3720,9 @@ package body Tree_Walk is
          when N_Pragma =>
             Do_Pragma (N, Block);
 
+         when N_Raise_Statement =>
+            null;
```

Comment:
Again, we should issue a warning when ignoring things just so we don't forget we're doing it.

Reply:
Or maybe a significant/highlighted comment like `FIXME` or `XXX`